src/animation.jl

```diff
@@ -77,6 +78,11 @@ mp4(anim::Animation, fn = mp4fn(); kw...) = buildanimation(anim, fn, false; kw..
 Creates an .webm-file from an `Animation` object.
 """
 webm(anim::Animation, fn = webmfn(); kw...) = buildanimation(anim, fn, false; kw...)
+"""
+    apng(animation[, filename]; fps=20, loop=0, variable_palette=false, verbose=false, show_msg=true)
```

Comment:
`variable_palette` is no used for `apng`s, please add only the used keyword args to the signature.